modules/cms/classes/Page.php

```diff
@@ -88,8 +89,24 @@ public function getLayoutOptions()
         $layouts = Layout::listInTheme($theme, true);
         $result = [];
         $result[null] = Lang::get('cms::lang.page.no_layout');
+        $ignoreNames = FileDefinitions::get('ignoreFiles');
+
         foreach ($layouts as $layout) {
             $baseName = $layout->getBaseFileName();
+
```

Comment:
I would prefer adding ignorePatterns to the filesystem definitions to make it more consistent with the rest of October, but I'll double check with @daftspunk.